include/RAJA/policy/sequential/scan.hpp

```diff
@@ -80,6 +80,7 @@ concepts::enable_if<type_traits::is_sequential_policy<ExecPolicy>>
 inclusive_inplace(const ExecPolicy &, Iter begin, Iter end, BinFn f)
 {
   auto agg = *begin;
+  RAJA_NoSIMD
```

Comment:
Please add a blank link  before each of the the RAJA_NoSIMD macros in this file to make it easier to visually parse which lines are variable declarations and which is the loop.